setup.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2019 CERN.
+#
+# CERN Analysis Preservation is free software; you can redistribute it
+# and/or modify it under the terms of the MIT License; see LICENSE file
+# for more details.
+from setuptools import setup
+
+tests_require = [
+    'isort>=4.3.4',
+    'coverage>=4.0',
+    'pydocstyle>=1.0.0',
+    'pytest-cache>=1.0',
+    'pytest-cov>=1.8.0',
+    'pytest-pep8>=1.0.6',
+    'pytest>=2.8.0',
+]
+
+extras_require = {
+    'tests': tests_require,
+}
+
+extras_require['all'] = []
+for key, reqs in extras_require.items():
+    if ':' == key[0]:
+        continue
+    extras_require['all'].extend(reqs)
+
+setup_requires = [
+    'pytest-runner>=2.7',
+]
+
+install_requires = [
+    'click>=7,<8',
+]
+
+setup(
+    name='twikiget',
+    version='0.0.1',
+    author='CERN Analysis Preservation',
+    py_modules=['wget'],
+    extras_require=extras_require,
+    install_requires=install_requires,
+    setup_requires=setup_requires,
+    tests_require=tests_require,
+    entry_points='''
```

Comment:
You can use the dictionary technique to specify entry points, as in `reana` main repository.  (Provided you introduce `twikiget` directory etc...)